client/python/pyproject.toml

```diff
@@ -3,7 +3,7 @@ name = "armada_client"
 version = "0.1.0-dev"
 description = "Armada gRPC API python client"
 readme = "README.md"
-requires-python = ">=3.7,<3.9"
+requires-python = ">=3.7,<3.9.13"
```

Comment:
Is there a reason this is using a minor version? If the desired behavior is py 3.9 support, IMO we should just make it `<3.10` so that we pick up py 3.9 minor releases for free.

Reply:
Resolved by kevins comment